contracts/test/MplRewards.t.sol

```diff
@@ -65,10 +65,10 @@ contract MplRewardsTest is DSTest {
     function test_notifyRewardAmount() external {
         uint256 totalRewards = 25_000 * WAD;
 
-        MplRewardsOwner owner      = new MplRewardsOwner();
-        MplRewardsOwner notOwner   = new MplRewardsOwner();
-        SomeERC2258 rewardsToken   = new SomeERC2258("RT", "RT");
-        MplRewards rewardsContract = new MplRewards(address(rewardsToken), address(0), address(owner));
+        Owner       owner           = new Owner();
+        Owner       notOwner        = new Owner();
+        SomeERC2258 rewardsToken    = new SomeERC2258("RT", "RT");
+        MplRewards  rewardsContract = new MplRewards(address(rewardsToken), address(0), address(owner));
```

Comment:
Alphabetical

Reply:
By I can do is
```suggestion
        Owner       notOwner        = new Owner();
        Owner       owner           = new Owner();
        SomeERC2258 rewardsToken    = new SomeERC2258("RT", "RT");

        MplRewards  rewardsContract = new MplRewards(address(rewardsToken), address(0), address(owner));
```
Since owner and rewardsToken are needed by MplRewards.